components/camel-telegram/src/main/docs/telegram-component.adoc

```diff
@@ -310,3 +310,60 @@ to("telegram:bots/123456789:AAE_dLq5C19xwGjw3yiC2NvEUrZcejK21-Q987654321:AAE_dLq
 ---------------------------------------------------------
 
 Note that the corresponding URI parameter is simply `chatId`.
+
+### Customizing keyboard
+
+You can customize the user keyboard instead of asking him to write an option. `OutgoingTextMessage` has the property `ReplyKeyboardMarkup` which can be used for such thing.
+
+[source,java]
+---------------------------------------------------------
+from("telegram:bots/123456789:AAE_dLq5C19xwGjw3yiC2NvEUrZcejK21-Q987654321:AAE_dLq5C19xwOmg5yiC2NvSrkT3wj5Q1-L")
```

Comment:
Please make sure you don't use a private bots, as the access key may be abused.

Reply:
I used the same access key available in the documentation. See other examples on it. This key is not mine.

Reply:
So it should be a random one I've generated (I hope :smile:)